interview-questions.md

```diff
@@ -7,60 +7,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: 
+    Git is a command prompt in terminal that interacts with data stored on Github. Github is a site that securely stores repositories and their branches so multiple can work on the same sections of code at the same time. 
 
+  Researched answer: 
+    Git is a command-line tool software installed locallay on the system that is used as a version control system to manage code history and code sharing. GitHub is a GUI service hosted on the web that is used to securely store Git repositories and centralized code. 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: 
+    Greater than ">", less than "<", stricly equals "===", and loose equals "==", bang operators used alongside strict equality "!==", greaterthan or equal to ">=" , and less than or equal to "<=". 
 
-  Researched answer:
+  Researched answer: 
+    Conditional statements. 
```

Comment:
Alright, I appreciate that you had a great answer above but use this space as opportunity to dive deep into the subjects. Next week I won't accept answers this short. 

I'll encourage you to share a time you used the tools in question or explain some aspect of them that you learned thats interesting.